src/Features/CSharp/Portable/MoveToResx/CSharpMoveToResxService.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.CSharp.MoveToResx;
+
+[DiagnosticAnalyzer(LanguageNames.CSharp)]
+internal class MoveToResx : DiagnosticAnalyzer
```

Comment:
Also, update filename to match.

```suggestion
internal class CSharpMoveToResxDiagnosticAnalyzer : DiagnosticAnalyzer
```

Reply:
also, seal the type.